test/plugin/scenarios/shenyu-2.4.x-scenario/shenyu-gateway/src/main/resources/shenyu-rule.json

```diff
@@ -0,0 +1,347 @@
+/**
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+*/
+{
+  "1512230008264994816": [
+    {
+      "id": "1512230008290160640",
```

Comment:
What IDs are these?

Reply:
This is the ID of the rule. It has no special meaning

Reply:
Would you replace them with `1,2,3` with comments to say this is just an ID for Shenyu setup? Or is this exported so it is like this?

Reply:
this is exported.  but now I have replace them with 1,2.  and deleted unnecessary and simplified configuration